docs/preview features/orchestrator.md

```diff
@@ -0,0 +1,136 @@
+# Orchestrator Preview
+Orchestrator is a powerful new language model that is optimized for conversational AI applications.
```

Comment:
"Orchestrator is a powerful LU solution optimized for conversational AI applications.".